src/metabase/api/collection.clj

```diff
@@ -105,9 +114,10 @@
 
 (defmethod fetch-collection-children :dashboard
   [_ collection {:keys [archived?]}]
-  (db/select [Dashboard :id :name :description :collection_position]
+  (-> (db/select [Dashboard :id :name :description :collection_position]
     :collection_id (:id collection)
-    :archived      (boolean archived?)))
+    :archived      (boolean archived?))
+      (hydrate :favorite)))
```

Comment:
You should reindent the args to `db/select` here now that you changed its indentation

Reply:
got any recs for vim indenters for this sort of thing?

Reply:
maybe @walterl has some pointers? (I know he uses vim for clojure)

Reply:
I've gotten used to tapping `==` (from [vim-sexp](https://github.com/guns/vim-sexp/#indent-commands-normal)) to format the current block, after leaving insert mode.